pkg/cmd/render/render.go

```diff
@@ -0,0 +1,76 @@
+package render
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/cluster-authentication-operator/bindata"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type Permission os.FileMode
```

Comment:
Why rename the std lib type? Use the original type instead. You'll avoid unnecessary casting when using `os` functions as well.

Reply:
I was using https://github.com/openshift/api/blob/4e32bc25f4f950292cc3a8b7322cba3407a62d2f/payload-command/render/renderassets/assets.go#L14-L20 as inspiration to try and stay consistent with how it is done elsewhere in the openshift org

I can update this to be the std lib type.